Dockerfile

```diff
@@ -61,6 +61,7 @@ RUN adduser -D -u 1000 semgrep
 USER 1000
 ENV SEMGREP_IN_DOCKER=1
 ENV SEMGREP_VERSION_CACHE_PATH=/tmp/.cache/semgrep_version
+ENV SEMGREP_USER_AGENT_APPEND="(Docker)"
```

Comment:
Should this be `None` so that we don't get faulty data in non-Docker environments?

Reply:
I'm not following. Can you elaborate? This only affects the Semgrep Docker image/container's environment, so there shouldn't be any faulty data elsewhere. In fact, it can even be overwritten at runtime with `docker run -e SEMGREP_USER_AGENT_APPEND=...`.

Reply:
Oh, I see -- this is in the Dockerfile 🤦‍♂️  I thought it was in the code itself. 👍